tests/sentry/api/endpoints/test_sentry_apps.py

```diff
@@ -135,48 +135,73 @@ def test_cannot_create_app_without_correct_permissions(self):
         response = self._post(**kwargs)
 
         assert response.status_code == 422
-        assert response.data['errors'] == \
+        assert response.data == \
             {'events': ['issue webhooks require the event:read permission.']}
 
+    @with_feature('organizations:internal-catchall')
+    def test_wrong_schema_format(self):
+        self.login_as(user=self.user)
+        kwargs = {'schema': {
+            'elements': [
+                {
+                    'type': 'alert-rule-action',
+                    'required_fields': [
+                        {
+                            'type': 'select',
+                            'label': 'Channel',
+                            'name': 'channel',
+                            'options': [
+                                ['#general'],
```

Comment:
Maybe add a comment here saying something like `# each option requires two values` so it's immediately obvious what's wrong with the schema, when we come back to this.